src/main/java/org/candlepin/controller/Entitler.java

```diff
@@ -234,9 +234,11 @@ public List<Entitlement> bindByProducts(AutobindData data, boolean force)
                 consumer.getUuid(), owner.getKey(), caMessage, hypMessage);
 
             if (autobindHypervisorDisabled) {
-                throw new AutobindHypervisorDisabledException(i18n.tr(
-                    "Auto-attach is disabled for owner \"{0}\"{1}.",
-                    owner.getKey(), hypMessage));
+                String message = owner.isAutobindHypervisorDisabled() ?
```

Comment:
Same deal here. The whole outer block is just a complicated way to avoid separate smaller blocks. We should just rewrite the whole thing rather than trying to work around the madness.

Reply:
Has there been any update or difficulties on this item?